src/main/cliapp/src/containers/diseaseAnnotationsPage/NewAnnotationForm.js

```diff
@@ -155,20 +166,22 @@ export const NewAnnotationForm = ({
 
 	const handleClear = () => {
 		//this manually resets the value of the input text in autocomplete fields with multiple values and the experiments dropdown
-		withRef.current.inputRef.current.value = "";
-		evidenceCodesRef.current.inputRef.current.value = "";
-		experimentsRef.current.clear();
+		if(withRef && withRef.current && withRef.current.inputRef && withRef.current.inputRef.current) withRef.current.inputRef.current.value = "";
```

Comment:
This could be done via `if(withRef?.current?.inputRef?.current)` it makes for a shorter line.